src/Standards/MySource/Sniffs/Commenting/FunctionCommentSniff.php

```diff
@@ -72,7 +72,7 @@ public function process(File $phpcsFile, $stackPtr)
             }//end if
         }//end foreach
 
-        if ($hasApiTag === true && substr($tokens[$tag]['content'], 0, 4) !== '@api') {
+        if ($hasApiTag === true && isset($tag) === true && substr($tokens[$tag]['content'], 0, 4) !== '@api') {
```

Comment:
Condition unnecessary, it will always be `true`.

`$hasApiTag` will only be `true` if the `foreach` loop has been entered and if so, `$tag` will be set.